apps/courses/src/routes/privacy-policy/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Heading,
+  Text,
+  List,
+  ListItem,
+  Divider,
+  Flex,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Icon,
+} from '@chakra-ui/core';
+import Page from '@openmined/shared/util-page';
+import content from '../../content/privacy-policy';
+import GridContainer from '../../components/GridContainer';
+
+import { Link, animateScroll as scroll } from 'react-scroll';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronUp, faCommentAlt } from '@fortawesome/free-solid-svg-icons';
+import CircledNumber from '../../components/CircledNumber';
+
+const SectionListItem = ({ content, index, onClick, ...props }) => (
+  <ListItem cursor="pointer" {...props}>
+    <Link
+      to={`accordion-button-${content.title
+        .replace(/\s+/g, '-')
+        .toLowerCase()}`}
+      onClick={() => onClick(index)}
+      smooth={true}
+      offset={-150}
+      duration={500}
+    >
+      <Flex alignItems="center">
+        <CircledNumber
+          backgroundColor="gray.800"
```

Comment:
You should be moving this component within this file, rather than separate, and therefore you shouldn't need to pass props at all.

Reply:
Reminder: add dark and light variants